src/main/java/org/cryptomator/cli/Args.java

```diff
@@ -79,14 +94,33 @@ public int getPort() {
 	}
 
 	public Set<String> getVaultNames() {
-		return vaultPaths.keySet().stream().filter(vaultPasswords::containsKey).map(String.class::cast).collect(Collectors.toSet());
+		Set<String> filteredVaults = vaultPaths.keySet().stream().filter(vaultPasswords::containsKey).map(String.class::cast).collect(Collectors.toSet());
```

Comment:
Yes. You are right. I am no (Java) software developer so this might not be the best code ;) I'll try with your improvements.